### PR TITLE
Add download logs action

### LINKED ISSUE #35: Add logs exporter
We should ideally add a simple button in grafana to download a single .tar containing logs for every container.
This will be very useful for debugging but most importantly for assistance to first adopters of the tool

### PR DESCRIPTION
**Pull request by Shourya742**
Closes #35 
1. Added loki and Promtail to get logs from all the required containers.
2. Added log-server to parse logs from containers via loki
3. Added Download Logs link in grafana dashboard

### CODE REVIEW

**Comment by GitGab19** [comment]
@Shourya742 could you add some lines in the README to mention this new cool feature and explain how to download logs and where to send them? I would redirect people to our `benchmarking-tool` channel in discord. Maybe we could also add this instructions into a README file generated by the log server which ends up being in the same .tar.
What do you think about this?

**Comment by Shourya742** [comment]
> @Shourya742 could you add some lines in the README to mention this new cool feature and explain how to download logs and where to send them? I would redirect people to our `benchmarking-tool` channel in discord. Maybe we could also add this instructions into a README file generated by the log server which ends up being in the same .tar. What do you think about this?

Let's do both. This way, if someone overlooks one, they can't miss the other.

File: docker-compose-config-c.yaml [hunk 1]
```diff
@@ -69,6 +78,7 @@ services:
     volumes:
       - bitcoin_pool_side_data:/root/.bitcoin
       - ./custom-configs/sri-roles/bitcoin-tp-pool.conf:/root/.bitcoin/bitcoin.conf
+      - ./logs:/var/logs
```
Comment (GitGab19) [inline docker-compose-config-c.yaml:81]: Why do you add this?

Comment (Shourya742) [reply docker-compose-config-c.yaml]: Nice catch !! Removed with ace6ff986dead2583668a87adf88b2189e72bfd1

**Review by GitGab19 — APPROVED** [review]

**Pull request merged**